common/reviews/api/webpack-embedded-dependencies-plugin.api.md

```diff
@@ -0,0 +1,45 @@
+## API Report File for "@rushstack/webpack-embedded-dependencies-plugin"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { Compiler } from 'webpack';
+import { IPackageJson } from '@rushstack/node-core-library';
+import { WebpackPluginInstance } from 'webpack';
+
+// @alpha
+class EmbeddedDependenciesWebpackPlugin implements WebpackPluginInstance {
+    constructor(options?: IEmbeddedDependenciesWebpackPluginOptions);
+    apply(compiler: Compiler): void;
+}
+export default EmbeddedDependenciesWebpackPlugin;
+
+// @alpha
+export interface IEmbeddedDependenciesWebpackPluginOptions {
+    generatedLicenseFilename?: LicenseFileName;
+    generateLicenseFile?: boolean;
+    generateLicenseFileFunction?: LicenseFileGeneratorFunction;
+    outputFileName?: string;
+}
+
+// @alpha
+export interface IPackageData extends IPackageJson {
+    author?: string | {
+        name?: string;
+    };
+    copyright: string | undefined;
+    licenses?: {
+        type: string;
+        url: string;
+    }[];
+    licenseSource?: string;
```

Comment:
Should we just include these fields in `IPackageJson` in `node-core-library`? @octogonz?

Reply:
I don't think `licenseSource` belongs in there personally but I'm apathetic to the others.